tests/test_cmdbase.py

```diff
@@ -87,3 +87,33 @@ class TestClass(BaseCommand):
     tc = TestClass("group", "config")
     with pytest.raises(NotImplementedError):
         tc.run([])
+
+
+# -- tests for strings in commands
+
+
+@pytest.mark.parametrize("command", all_commands)
+def test_aesthetic_help_msg(command):
+    """All the real commands help msg start with uppercase and doesn't end with a dot."""
```

Comment:
This looks more like an integration test.

```suggestion
    """All real command help msgs start with uppercase and do not end with a dot."""
```